src/components/Books.js

```diff
@@ -1,28 +1,33 @@
 import React from 'react';
+import { useDispatch } from 'react-redux';
 import PropTypes from 'prop-types';
+import { removeBook } from '../redux/books/books';
 
-const Books = (props) => {
-  const {
-    book: {
-      title,
-      author,
-      id,
-      category,
-    },
-
-  } = props;
+const Books = ({ book }) => {
+  const [id, [item]] = book;
+  const dispatch = useDispatch();
 
   return (
-    <>
-      <li key={id} className="books-content">
-        {title}
-        {' by '}
-        {author}
-        {' - '}
-        {category}
-      </li>
-      <button type="button" id={id}>Remove</button>
-    </>
+    <div className="books-contents">
+      <div className="first">
+        <li key={id} className="book-cont">
+          <span className="category">{item.category}</span>
+          <span className="title">{item.title}</span>
+          <span className="author">{item.author}</span>
+        </li>
+        <div className="buttons">
+          <button type="button" className="btn border"> Comment</button>
+          <button type="button" className="btn border" id={id} onClick={() => dispatch(removeBook(id))}>Remove</button>
+          <button type="button" className="btn">Edit</button>
+        </div>
+      </div>
+
+      <div className="book-ch">
+        <span className="current-chapt">CURRENT CHAPTER</span>
+        <span className="chapt">Chapter 17</span>
+        <button type="button" className="update-btn">UPDATE PROGRESS</button>
+      </div>
+    </div>
   );
 };
 
```

Comment:
- [ ] Great job creating the app however there are some errors on the console, please consider fixing them first. so your app has no errors.
   - In the prop validation, you are checking for an object, but in the props, you're passing an array,
   - also there are duplicated keys on your array, so you can use Math.random() or use the UUID to generate a random key

![Screenshot_20220617_004832](https://user-images.githubusercontent.com/25227022/174196323-54e342d8-f60e-4d56-bebe-a3759b1ba03a.png)